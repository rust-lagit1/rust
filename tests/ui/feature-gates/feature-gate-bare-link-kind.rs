#[link(kind = "dylib")] //~ ERROR `#[link]` attribute requires a `name = "string"` argument
extern "C" {
    static FOO: u32;
}

fn main() {}
