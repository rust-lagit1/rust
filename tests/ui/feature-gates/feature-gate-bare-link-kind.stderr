error[E0459]: `#[link]` attribute requires a `name = "string"` argument
  --> $DIR/feature-gate-bare-link-kind.rs:1:1
   |
LL | #[link(kind = "dylib")]
   | ^^^^^^^^^^^^^^^^^^^^^^^ missing `name` argument

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0459`.
